rest/data/interface.go

```diff
@@ -236,4 +238,7 @@ type Connector interface {
 	CommitQueueRemoveItem(string, string) (bool, error)
 	CommitQueueClearAll() (int, error)
 	IsAuthorizedToPatchAndMerge(context.Context, *evergreen.Settings, UserRepoInfo) (bool, error)
+
+	//Docker methods
```

Comment:
This should be a comment of the form "GetLogs...". You could add an additional comment separating the Docker methods. Alternatively, following John's lead about `GetClient`, since this is Docker-specific, you could name this `GetDockerLogs`. (You could imagine in the future another function that gets logs from a different type of container.)